patterns/2-structured/praise-participants.md

```diff
@@ -80,5 +81,4 @@ Overdoing it may feel insincere and mechanical and defeat your purpose in reachi
 
 ## Acknowledgments
 
-* [Todd Lisonbee](https://github.com/tlisonbee) for encouraging to "keep it real".
-* [Isabel Drost-Fromm](https://github.com/MaineC) for [this extra explanation](https://youtu.be/h3MPewsk5PU?t=357) of a "qualified" thank you.
```

Comment:
That's correct. Because I removed the word "qualified," I removed this reference to it.

Reply:
I will leave the decision about whether to remove this up to you. Obviously it is not doing any harm if it remains in place, but because the word "qualified" is removed in the preceding, we'll need to re-word the footnote a bit here.

Reply:
I have added it back in and reworded it.